db/config.go

```diff
@@ -28,8 +28,19 @@ type Config struct {
 	HistoryStateRetention uint64 `yaml:"historyStateRetention"`
 	// ReadOnly is set db to be opened in read only mode
 	ReadOnly bool `yaml:"readOnly"`
+	// DBType is the type of database
+	DBType Type `yaml:"dbType"`
 }
 
+// Type is the type of database
+type Type string
+
+// Database types
+const (
+	DBBolt   Type = "bolt"
+	DBPebble      = "pebble"
+)
```

Comment:
think can omit this, just use `string`  "Bolt", "Pebble", straightforward and very clear